package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "This Leaderboard app uses an API to receive and display the scores of the game of different players. It is built with HTML, CSS and JavaScript.",
+  "main": "webpack.config.js",
+  "scripts": {
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^6.8.1",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "hint": "^7.1.10",
+    "style-loader": "^3.3.3",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack-cli": "^5.1.4"
```

Comment:
- You have done great so far :+1: :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :+1: 
- [ ] Currently, you do not have the `webpack-dev-server` in your project, and its needed to open your project server in the browser, I'll suggest you add this to your devDepndencies by running this command  `npm install webpack-dev-server` :+1: :wave: